week_6/week_6_quiz.md

```diff
@@ -0,0 +1,251 @@
+
+
+**Question 2**
+
+The goals of the ABCD substance use module include the characterization of: (check all that apply)
+
+- [ ] risk and resilience factors for substance use and SUD development (e.g., early substance use attitudes, expectancies, subjective effects, peer use, parental rules)
+- [ ] the impact of polysubstance use patterns and “gateway interactions” between different substances
+- [ ] the consequences of repeated substance use on neurocognitive, emotional, and physical development
+- [ ] associations between substance use and psychopathology
+- [ ] differential trajectories of substance use initiation, experimentation, and use patterns
+
+
+**Question 3**
+
+Biospecimens (e.g., saliva, hair, and urine samples) provide a more objective measure of recent substance use and exposure. Biospecimen samples are collected for ALL participants enrolled in the ABCD Study at each in-person visit.
+
+- True
+- False
+
+**Question 4**
+
+Which of the following are examples of gating in the ABCD substance use interview? (check all that apply)
+
+- [ ] In years baseline-2 year follow-up, a participant indicates they have “heard of” a given substance, additional questions about that particular substance will follow
+- [ ] If a participant indicates they have heard of a potential “gateway” substance, additional questions about other substances will follow
+- [ ] If a participant indicates they have NOT used a given substance, zeros may be inserted for any follow-up questions triggered by this question
+- [ ] If a participant has indicated they have used a given substance, they will be asked to recall details of their first experience with that substance at future visits
+
+
+**Question 5**
+
+The Timeline Follow Back asks about detailed month-by-month substance use (i.e., date of use, amount in standard units) and applicable follow-up questions for some substances such as cannabis (e.g., route of administration) for the 12-month period preceding the interview date.
+
+- True
+- False
+
+
+### ReproNim NIDM data questions
+
+#### Prerequisites
+
+1. You have already [forked](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/fork-a-repo) and [cloned](https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/cloning-a-repository): https://github.com/ABCD-ReproNim/sample_dataset
+2. You have installed [pynidm](https://pypi.org/project/pynidm/) and [pyontutils](https://pypi.org/project/pyontutils/) (or are using our jupyterhub)
+    - [notes on running jupyterhub locally for observer students](https://neurostars.org/t/using-abcd-repronim-jupyterhub-container-locally-via-docker/17439)
+
+**Question 6**
+
+How does structured data annotation help the research process? (Select all that apply)
+
+- [ ] Can make filtering information within and between datasets easier
+- [ ] Can create less ambiguity about the measures recorded in a dataset
```

Comment:
"Can reduce ambiguity" might be a little easier to interpret.

Reply:
thanks, changed wording.